src/projects/TodoList/components/CategoryCheckBoxes/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './style.css';
+
+class CategoryCheckBoxes extends React.Component {
+  render () {
+    // logic goes here
+    let eachCategory = this.props.categoriesInfo.map((category, i) => {
+      let categoryBoxColour = {
+        backgroundColor: category.colour,
+        color: category.text
+      };
+
+      return (
+        <span className="checkbox-wrapper" style={categoryBoxColour} key={i}>
```

Comment:
You should replace this `<span>` tag with a `<label>` tag, its much better suited for inputs because with a label you can have it so if you click the label it also clicks the input (much better UX)

[The docs for label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) have a really good example at the top which illustrates the difference between using and not using a label

Reply:
very good point. ill do that now

Reply:
done